odoo-shippable/scripts/build-image.sh

```diff
@@ -419,16 +419,13 @@ usermod -s /bin/bash root
 for BASHRC in ${HOME}/.bashrc /home/odoo/.bashrc ${HOME}/.zshrc /home/odoo/.zshrc
 do
     echo "Export the PYTHONPATH IN ${BASHRC}"
-    sed -i '1 i\# Python path \
-if [ "x${TRAVIS_PYTHON_VERSION}" == "x"  ] ; then \
-    TRAVIS_PYTHON_VERSION="2.7" \
-fi \
-source /.repo_requirements/virtualenv/python${TRAVIS_PYTHON_VERSION}/bin/activate \
-source /.repo_requirements/virtualenv/nodejs/bin/activate \
-PYTHONPATH=${PYTHONPATH}:${REPO_REQUIREMENTS}/odoo\n\n' $BASHRC
 	cat >> $BASHRC << EOF
+if [ "x\${TRAVIS_PYTHON_VERSION}" == "x"  ] ; then
```

Comment:
you need scape $?

Reply:
hello?

Reply:
@moylop260 
I just copy and paste the follow commit https://github.com/Vauxoo/docker-odoo-image/blob/611fdba993b8764f07cbf16323a71974c53f5ff5/odoo-shippable/scripts/build-image.sh

This commit is previous to my PR https://github.com/Vauxoo/docker-odoo-image/pull/278